### PR TITLE
Update task.md

### PR DESCRIPTION
**Pull request by fredlee12001**
Add the links to sub-groups. Follow the similar way as other API page.

### CODE REVIEW

**Comment by BlackstoneEngineering** [comment]
@iriark01 @AnotherButler

**Comment by AnotherButler** [comment]
@iriark01 The problem with my link changes is they only go to 5.3 documents. Is there a way to include URLs that always go to the latest version?

**Comment by BlackstoneEngineering** [comment]
@iriark01 How can we provide /tip or /latest links to docs? Right now the URL links can only be absolute links, thus creating alot of technical debt for each doc needing to be updated for each release. Need a method of pointing to the latest docs.

**Comment by iriark01** [comment]
We don't have it right now because of a technical issue.

**Comment by iriark01** [comment]
You can relax. I'm taking care of it, these things don't get fixed in 30 seconds no matter how many blockers you file.

**Comment by BlackstoneEngineering** [comment]
whats the ETA? 2 days, 2 weeks 2 months...?

**Comment by iriark01** [comment]
It is partially fixed (you can already use https://docs.mbed.com/docs/mbed-os-handbook/en/latest/ and https://docs.mbed.com/docs/mbed-os-api-reference/en/latest/) but it has weird side-effects on the behaviour of the docs. Nothing you'll care about, I think.

**Comment by iriark01** [comment]
But I would like to point out that links to docs in the same project can be relative, so they shouldn't have the whole https://docs.mbed.com/docs/mbed-os-api-reference/en/latest/ bit in front of them.

File: docs/APIs/tasks/tasks.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
...
+The task management APIs handle creation and destruction of threads in mbed OS 5, as well as mechanisms for safe interthread communication. Threads are a core component of mbed OS 5 (even your `main` function starts in a thread of its own), so understanding how to work with them is an important part of developing applications for mbed OS 5. Pay particular attention to the [interrupt service routines](rtos.md#interrupt-service-routines) section, which contains important information about how the task management APIs can be used from an interrupt handler. 
+
+* [RTOS](https://docs.mbed.com/docs/mbed-os-api-reference/en/5.3/APIs/tasks/rtos/): The CMSIS-RTOS core.
```
Comment (iriark01) [inline docs/APIs/tasks/tasks.md]: This is all in the same project; you don't need the full URL, only the relative URL (which in this case is rtos.md)

**Comment by iriark01** [comment]
Why do you think you can't use relative links?

**Comment by BlackstoneEngineering** [comment]
talked with irit, I misunderstood. You can use relative links within a repo, but not between repos.

**Pull request merged**